src/styles/main_page/_Timeline.scss

```diff
@@ -0,0 +1,82 @@
+@import "../global_vars";
+
```

Comment:
Maybe you want to put these into an outer container so that it does not cause conflicts.